networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -22,7 +22,7 @@ image::527-OpenShift-UDN-isolation-012025.png[Namespace isolation concept in a u
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
 ====
 
-Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR. 
+Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR.
```

Comment:
a cluster administrator can use **CUDN** to create and define additional networks that span across multiple namespaces at cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR).

Use CUDN as you are describing UDN in the following sentence, it being namespace scoped.

Reply:
That update is in a different pr

Reply:
Recommending to use CUDN because with UDN user cannot create a 'user defined network' across multiple namespaces.

Reply:
This the difference between UDN as a general use which refers to both UDN and CUDN CR. We are saying what you are saying just using user-defined network in a general sense. 

Either way, this is just trailing whitespace that is being cut out nothing is an actual change to existing docs.

Reply:
OK.